README.md

```diff
@@ -8,21 +8,37 @@ Ensure that only a single instance of a process is running in your Kubernetes cl
 capabilities to coordinate commands running in different pods. It acts as a gatekeeper, only
 starting the command when the pod becomes a leader. 
 
+### Why leader election?
+Some applications aren't natively able to run in a replicated way.
+For example, they expect exclusive access to a shared resource (i.e. a database).
```

Comment:
"i.e." doesn't really fit here. Perhaps "typically".

In fact, in the case of the linstor controller, it isn't really the database, but the fact that it maintains a lot of internal state in memory.